<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scanner Camera Controls</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="gridFrame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <widget class="QLineEdit" name="PageNoL">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">padding-left:10px</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="PageNoR">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">padding-right: 10px</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="camPreviews">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="scanBtn">
      <property name="text">
       <string>Scan Now</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDirectory">
    <property name="title">
     <string>Directory</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionSwitch_Cameras"/>
    <addaction name="actionSync"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionArabic"/>
    <addaction name="actionRoman_Upper"/>
    <addaction name="actionRoman_Lower"/>
    <addaction name="actionPadding_Character"/>
    <addaction name="actionPadding_Length"/>
    <addaction name="actionFile_Name_Prefix"/>
    <addaction name="actionFile_Name_Suffix"/>
   </widget>
   <addaction name="menuDirectory"/>
   <addaction name="menuAction"/>
   <addaction name="menuFormat"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSwitch_Cameras">
   <property name="text">
    <string>Switch Cameras</string>
   </property>
  </action>
  <action name="actionArabic">
   <property name="text">
    <string>Arabic</string>
   </property>
  </action>
  <action name="actionRoman_Upper">
   <property name="text">
    <string>Roman Upper</string>
   </property>
  </action>
  <action name="actionRoman_Lower">
   <property name="text">
    <string>Roman Lower</string>
   </property>
  </action>
  <action name="actionPadding_Character">
   <property name="text">
    <string>Padding Character</string>
   </property>
  </action>
  <action name="actionPadding_Length">
   <property name="text">
    <string>File Name Length</string>
   </property>
  </action>
  <action name="actionFile_Name_Prefix">
   <property name="text">
    <string>File Name Prefix</string>
   </property>
  </action>
  <action name="actionFile_Name_Suffix">
   <property name="text">
    <string>File Name Suffix</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>Sync</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>scanBtn</tabstop>
  <tabstop>PageNoR</tabstop>
  <tabstop>PageNoL</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
